### PR TITLE
Typing menu elements.

### PR DESCRIPTION
**Pull request by vnmabus**

### CODE REVIEW

File: tuxemon/menu/interface.py [hunk 1]
```diff
@@ -1,33 +1,50 @@
+from __future__ import annotations
 import pygame
 
 from tuxemon import tools, graphics
 from tuxemon.ui.draw import GraphicBox
+from typing import ClassVar, Optional, Callable
 
 
-class Bar:
+class Bar():
```
Comment (bitcraft) [inline tuxemon/menu/interface.py]: why are the parens added?

Comment (vnmabus) [reply tuxemon/menu/interface.py]: Sorry, I always use them as for me that syntax is more consistent with other class and function definitions. I can remove them if you prefer.

Comment (bitcraft) [reply tuxemon/menu/interface.py]: the need for parens was removed with python3 for new-style classes and its pretty unusual to see in modern code.  i can't find anything official that says to leave them out, but the docs do show the class def using the form without parens.
https://docs.python.org/3/tutorial/classes.html#class-definition-syntax

Comment (vnmabus) [reply tuxemon/menu/interface.py]: Removed.

**Review by bitcraft — APPROVED** [review]

**Pull request merged**